CMakeLists.txt

```diff
@@ -34,6 +34,10 @@ CHECK_COMPILERS()
 find_package(Boost REQUIRED COMPONENTS thread filesystem system regex)
 find_package(GTest REQUIRED)
 find_package (Python3 COMPONENTS Interpreter Development)
+find_library(NUMA_LIBRARY NAMES numa)
+if (NOT NUMA_LIBRARY)
```

Comment:
does `find_package`  + `REQUIRED` not work here?

Reply:
I tried this locally and see the assert being hit. Looked online and I think for `find_package` to work on Numa, we need a `FindNuma.cmake` file.